src/simtools/corsika/corsika_config.py

```diff
@@ -124,9 +124,12 @@ def fill_corsika_configuration(self, args_dict, db_config=None):
         if db_config is None:  # all following parameter require DB
             return config
 
-        db_model_parameters = ModelParameter(
-            mongo_db_config=db_config, model_version=args_dict["model_version"]
-        )
+        model_version = args_dict.get("model_version", None)
+        # If the user provided multiple model versions, we take the first one
```

Comment:
Do we need a test that the corsika configs are actually the same for all model_versions? It is likely to be the same, but we can't be sure, or?

Reply:
Apparently we do need it, yes. It helped me find an error in the CORSIKA config of Prod5. I therefore added a function that asserts they are the same for parameters which are relevant. I doubt we will ever see a mismatch in the future, but it doesn't seem to hurt to check it (thanks to the cache).